test/inputs/text-input.spec.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var expect = require('chai').expect;
+var TextInput = require('../../lib/inputs/text-input');
+var expectError = require('./input-test-utils').expectError;
+
+describe('text input', () => {
+    describe('valid cases', () => {
+        const testObj = {
+            id: 'id',
+            type: 'text',
+            value: 'default value',
```

Comment:
Change `value` to be different from `params.default` to make sure `params.default` doesn't override `value`.